lib/oidc/util/loginRedirect.ts

```diff
@@ -31,6 +37,16 @@ export function isRedirectUri(uri: string, sdk: OktaAuth): boolean {
   return uri && uri.indexOf(authParams.redirectUri) === 0;
 }
 
+export function isCodeFlow(options: OktaAuthOptions) {
+  return options.pkce || options.responseType === 'code' || options.responseMode === 'query';
+}
+
+export function getHashOrSearch(options: OktaAuthOptions) {
```

Comment:
nit: getSearchOrHash()?   (which does it default to?)

Reply:
The default depends on your configured responseType and pkce setting. implicit is hash, authorization code is query.